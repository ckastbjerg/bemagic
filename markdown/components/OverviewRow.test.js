/* eslint-env jest */
const OverviewRow = require('./OverviewRow');

test('it throws', () => {
    expect(() => OverviewRow()).toThrow();
});

test('it works', () => {
    expect(OverviewRow({
        tagName: 'div',
        classes: 'block',
        states: ['is-active', 'is-dimmed'],
        type: 'component',
    })).toMatchSnapshot();
});
