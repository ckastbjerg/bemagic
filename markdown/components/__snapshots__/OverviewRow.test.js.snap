// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
"| **[\`block\`](#block)** | [component](../CONVENTIONS.md#components) | \`is-active\` and \`is-dimmed\`. | div |
"
`;
